### PR TITLE
Lighthouse CI

### PR DESCRIPTION
**Pull request by Wawrzyn321**
**Description**

Changes proposed in this pull request:

- Remember to copy a kubeconfig to `fixtures/kubeconfig.yaml`

**Related issue(s)**

Connected #982
[test-infra](https://github.com/kyma-project/test-infra/pull/5101)

### CODE REVIEW

File: lighthouse/playwright.config.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  projects: [
+    {
+      name: 'chromium',
+      use: {
+        ...devices['Desktop Chrome'],
+      },
+    },
+
+    // {
+    //   name: 'firefox',
```
Comment (dariadomagala) [inline lighthouse/playwright.config.js:54]: can we also add scans for firefox, or does it need additional configurations?

File: lighthouse/lighthouse.spec.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+  await expect(
+    page.frameLocator('iframe').locator('text=Cluster Overview'),
+  ).toBeVisible();
+
+  await playAudit({
+    page,
+    port: 9222,
+    thresholds: {
+      performance: 3,
```
Comment (dariadomagala) [inline lighthouse/lighthouse.spec.js:59]: Can we set higher values here? Checking if it's higher than 3 is not valuable at all, and on my machine, the performance value is only 5, which is also pretty far from what the lighthouse shows normally.

File: lighthouse/lighthouse.spec.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+test('Busola Lighthouse audit', async () => {
+  const context = await chromium.launchPersistentContext(tmpdir(), {
+    args: ['--remote-debugging-port=9222'],
+  });
+  const page = await context.newPage();
+
+  await page.goto('http://localhost:8080');
+
+  // on dev chromium may remember previous state - get back to clusters list
+  if (!page.url().endsWith('/clusters')) {
```
Comment (dariadomagala) [inline lighthouse/lighthouse.spec.js:15]: it doesn't work on a first run. Address is just `localhost:8080`, and only after a while it gets redirected to `.../clusters`.

Comment (Wawrzyn321) [reply lighthouse/lighthouse.spec.js:15]: Woah, I forgot that /clusters actually **doesn't** redirect you to current cluster... Fixed!

Comment (Wawrzyn321) [reply lighthouse/lighthouse.spec.js:15]: Or actually... no, it does redirect to current cluster. I don't know why.

File: lighthouse/playwright.config.js (hunk 1 above)
Comment (Wawrzyn321) [reply lighthouse/playwright.config.js:54]: Nope, Lighthouse is Chrome(chromium?)-specific.

Comment (Wawrzyn321) [reply lighthouse/playwright.config.js:54]: I mean, there's a Lighthouse extension for Firefox

**Review by dariadomagala — APPROVED** [review]

**Review by grego952 — APPROVED** [review]

**Pull request merged**